recursion/recursionFibonacci/FibonacciPrinter.go

```diff
@@ -0,0 +1,5 @@
+package FibonacciElements
+
+type FibonacciPrinter interface {
```

Comment:
I think the interface name should tell what it is doing. Since it is returning fibonacci rather than printing ... so you name like FibonacciComputer or like that

Reply:
fixed